Official Warp Extension

joetannenbaum opened the pull request:
## Description

Added commands:

- Open Directory
- Open Selected Finder Item
- Open Launch Configuration
- New Tab
- New Window

## Screencast

Included screenshots of relevant commands in the `metadata` directory.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

peduarte reviewed (approved):
Very nice 👏 

I have some non-blocking comments:
- I personally don't see any value in the _New Window_ and _New Tab_ commands. Maybe I'm missing something, but I find much easier to ⌘N and ⌘T 
- The _Open Selected Finder Item_ command is great! But I think I'd rename it to _Open Folder In Warp_ or _Open Finder Folder in Warp_ or even simply _Open in Warp_
  - On this note, would be nice if the current finder path isn't a directory, instead of showing "no directory selected" to open the parent directory

Let me know if you'd like to make any of the changes above, or if you're happy to release as is.

Thanks for your contribution!

joetannenbaum commented:
@peduarte thank you!

I tend to agree with you, I don't think they serve much value, but the team wanted feature parity with other terminal Raycast extensions so I added it in.

Good call on simplifying the command name, I changed it to "Open in Warp", makes more sense. I created a fallback for when no items are explicitly selected in the Finder, it should now open the parent directory by default. Let me know if that looks right to you.

I also added the ability to add a launch config as a quicklink in this change set.

peduarte commented:
Makes sense, love the changes! 🚀

peduarte commented on extensions/warp-launch-configuration/CHANGELOG.md:
```diff
@@ -1,11 +1,9 @@
-
-## [Initial Version] - 2023-03-07
+- "Open Directory" command
+- "Open Selected Finder Item" command
```
```suggestion
- "Open in Warp" command
```

peduarte commented:
I updated the changelog. Will wait for build to pass then we release.
